src/dht/mod.rs

```diff
@@ -180,15 +179,8 @@ async fn new(
     identity_store: Arc<dyn IdentityStoreApi>,
     file_upload_store: Arc<dyn FileUploadStoreApi>,
 ) -> Result<(Client, Receiver<Event>, EventLoop)> {
-    if !identity_store.libp2p_credentials_exist().await {
-        let keys = BcrKeys::new();
-        let p2p_keys = keys.get_libp2p_keys()?;
-        let node_id = p2p_keys.public().to_peer_id();
-        identity_store.save_node_id(&node_id).await?;
-        identity_store.save_key_pair(&keys).await?;
-    }
-
-    let local_public_key = identity_store.get_key_pair().await?.get_libp2p_keys()?;
+    let keys = identity_store.get_or_create_key_pair().await?;
```

Comment:
what key is it? SecP or Rsa? Maybe better name for this varuable

Reply:
secP in this case - in new code, I used `key`, `private_key` and `public_key` for our "main" keys (secp) and if it's rsa, I tried to call them `rsa_keys`, `rsa_public_key` etc. as a convention.

This would be consistent with that, but we'd have to change it once across the whole codebase to be really clean.